### ISSUE TITLE
Trailing slash appended to robots.txt and sitemap.xml

### ISSUE THREAD

**Issue by leanderiversen** [sst/open-next#286]
Hi again,

I have found a new issue regarding trailing slashes, and can confirm that it is present in at least `2.2.3` and `2.2.4`.  The `robots.txt (app/robots.ts)` and `sitemap.xml (app/sitemap.ts)` files get a trailing slash appended, when they were previously left alone.

Happy to provide more details if required.

Thank you.

**Comment by khuezy** [comment]
Are these special files that shouldn't have a trailing slash, even if `trailingSlash` is true?

**Comment by leanderiversen** [comment]
@khuezy That is correct. They should "live" at root level, and not have a trailing slash, even if `trailingSlash` is true.

The documentation can be found [here](https://nextjs.org/docs/app/api-reference/file-conventions/metadata/sitemap) and [here](https://nextjs.org/docs/app/api-reference/file-conventions/metadata/robots).

**Comment by khuezy** [comment]
Cool thanks, are there any other special files that need to behave like this?

**Comment by leanderiversen** [comment]
It looks like there is [app/manifest.ts](https://nextjs.org/docs/app/api-reference/file-conventions/metadata/manifest) as well, but I think that is the complete list.
